packages/tupaia-web/src/constants/constants.ts

```diff
@@ -37,3 +37,15 @@ export const FORM_FIELD_VALIDATION = {
 };
 
 export const PASSWORD_RESET_TOKEN_PARAM = 'passwordResetToken';
+
+export enum TABS {
+  MAP = 'map',
+  DASHBOARD = 'dashboard',
+}
+
+export const MOBILE_THRESHOLD = '900px';
```

Comment:
Can you just add a comment explaining what this is. and What do you think about the name MOBILE_LAYOUT_BREAKPOINT? or MOBILE_BREAKPOINT?